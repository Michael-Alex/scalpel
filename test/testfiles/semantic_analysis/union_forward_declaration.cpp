union u;
union u;
union u
{
};

class c
{
	union u;
};

union c::u
{
};

