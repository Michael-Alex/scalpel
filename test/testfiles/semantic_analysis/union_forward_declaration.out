{
  "id": "namespace-0",
  "classes":
  [
    {
      "id": "class-0",
      "name": "c",
      "unions":
      [
        {
          "id": "member-union-0",
          "name": "u",
          "access": "private",
          "destructor":
          {
            "access": "public"
          }
        }
      ],
      "destructor":
      {
        "access": "public"
      }
    }
  ],
  "unions":
  [
    {
      "id": "union-0",
      "name": "u",
      "destructor":
      {
        "access": "public"
      }
    }
  ]
}
